runtime/astar/src/lib.rs

```diff
@@ -807,19 +807,23 @@ impl pallet_evm::Config for Runtime {
     type ChainId = ChainId;
     type OnChargeTransaction = pallet_evm::EVMCurrencyAdapter<Balances, ToStakingPot>;
     type BlockGasLimit = BlockGasLimit;
+    type Timestamp = Timestamp;
     type OnCreate = ();
     type FindAuthor = FindAuthorTruncated<Aura>;
     type WeightInfo = pallet_evm::weights::SubstrateWeight<Runtime>;
 }
 
 parameter_types! {
     pub const PostBlockAndTxnHashes: PostLogContent = PostLogContent::BlockAndTxnHashes;
+    // Maximum length (in bytes) of revert message to include in Executed event
+    pub const ExtraDataLength: u32 = 30;
```

Comment:
Is this number from the template? Do we understand what's the potential impact of this?

Github search tells me that e.g. Darwinia is using 64.

Reply:
Yes, the number is from template. I was being cautious and decided for value this low. Locally testing this does not give much input regarding load.

Impact - The larger the value, more memory it'll take for allocating bytes for revert message.

btw 30 bytes (30 character, assuming ascii) is like really low.